android-components/components/support/ktx/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,11 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mozilla.components.support.ktx" />
+    package="mozilla.components.support.ktx">
+    <queries>
```

Comment:
This was added in order to call queryIntentActivities

<img width="1292" alt="Screenshot 2022-11-03 at 13 42 56" src="https://user-images.githubusercontent.com/32488956/199719477-47396196-78ec-4bdc-ba2a-e54e0bddf38f.png">

Reply:
Does this mean we can remove the `<queries>` declaration from `ContextMenu` feature?